### PR TITLE
Release prep v8.0.0

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,53 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v8.0.0](https://github.com/puppetlabs/puppetlabs-apache/tree/v7.0.0) (2022-08-05)
```
Comment (petergmurphy) [inline CHANGELOG.md:5]: The v7.0.0 in the link should be v8.0.0

**Review by ekohl — COMMENTED** [review]
I wonder if we should finish https://github.com/puppetlabs/puppetlabs-apache/pull/2277 before releasing this. PHP is quite commonly used and a clear error when trying to use it is better IMHO.

File: CHANGELOG.md (hunk 1 above)
Comment (david22swan) [reply CHANGELOG.md]: fixed

**Comment by david22swan** [comment]
PR updated to include a ru of the REFERENCE.md generate command:
```
bundle exec puppet strings generate --format markdown --out REFERENCE.md
```

**Comment by david22swan** [comment]
> 

It would probably be better, and you do not seem far away from a fix.....
Yeh, I'm good with waiting

**Comment by ekohl** [comment]
> It would probably be better, and you do not seem far away from a fix.....

The tests are running, if they turn green I think it should be good.

**Comment by david22swan** [comment]
Will re-open when the php fix goes in

**Pull request closed without merging**